Porting/pod_lib.pl

```diff
@@ -405,7 +405,7 @@ sub __prime_state {
     my $filename = "pod/$source";
     my $contents = slurp_or_die($filename);
     my @want =
-        $contents =~ /perldelta - what is new for perl v(5)\.(\d+)\.(\d+)\r?\n/;
+        $contents =~ /perldelta - what is new for perl v([57])\.(\d+)\.(\d+)\r?\n/a;
```

Comment:
Isn't this going to cause the same problem when we get to perl 8? How about just `\d+` like the other components?

Reply:
yes was considering `\d+`, I think this is a better choice, fixed by 5169f4d8